app/Models/Application.php

```
@@ -1555,40 +1555,206 @@ protected function buildGitCheckoutCommand($target): string
+
+            // If no patterns matched and we only have exclusion patterns, include by default
+            if ($shouldInclude === null) {
+                // Check if we only have exclusion patterns
+                $hasInclusionPatterns = $watch_paths->contains(fn ($p) => ! str_starts_with(trim($p), '!'));
+
+                return ! $hasInclusionPatterns;
+            }
+
+            return $shouldInclude;
+        })->values();
+    }
```

PHPMD (2.15.0)

The parameter $modified_files is not named in camelCase. (undefined)

(CamelCaseParameterName)